### DISCUSSION TITLE
update metainfo file for aug11 to au57x

### DISCUSSION THREAD

**Discussion by willblk973** [Mr-MIBonk/M.I.B._More-Incredible-Bash#84] [Ideas]
Hello, I have the train MHI2_US_AUG11_S0533. Is it possible for you to create a meta file for me to go to MHI2_US_AU57x_K3341_MU1426. Or give instruction on how to do it.

**Comment by harman-f** [comment]
There is currently a similar discussion going on here:
https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/discussions/82

Please continue there.

Agree on a FW version and I can provide a modified metainfo.
